data/test/scenarios/ai/config/simple_aspect.cfg

```diff
@@ -1,48 +1,48 @@
 
-{AI_ASPECT_UNIT_TEST test_basic_simplified_aspect attack_depth
-{SPLIT 3,5,5,3,7,7,5,3,5,5,7,7 ,}
+{AI_ASPECT_UNIT_TEST test_basic_simplified_aspect leader_value
+{SPLIT 9,3,3,9,7,7,3,9,3,3,7,7 ,}
```

Comment:
I don't understand why the test numbers were changed? Was there a reason for it? I mean I guess it doesn't matter as long as the test still tests what it's supposed to test, but...

Reply:
The change of numbers is needed because the default values for `attack_depth` and `leader_value` are different.